examples/use-terraformer-instance/README.md

```diff
@@ -0,0 +1,80 @@
+# Using the Terraform instance #
+
+This example code shows how to use the Terraformer code to spin up a
+new instance in the operations subnet of an existing assessment
+environment already created by running `terraform apply` with the
+[cisagov/cool-assessment-terraform](https://github.com/cisagov/cool-assessment-terraform)
+root module.
+
+## Requirements ##
+
+No requirements.
```

Comment:
Are we intentionally leaving out a `versions.tf` file for this example?

Reply:
Please see commit a22bb64.